main.go

```diff
@@ -377,6 +379,20 @@ func Flash(pkgName, port string, options *compileopts.Options) error {
 				return &commandError{"failed to flash", result.Binary, err}
 			}
 			return nil
+		case "bmp":
+			bmpDebugPort := "/dev/ttyACM0"
+			if len(port) > 0 {
+				bmpDebugPort = port
+			}
+			args := []string{"-ex", "target extended-remote " + bmpDebugPort, "-ex", "monitor swdp_scan", "-ex", "attach 1", "-ex", "load", filepath.ToSlash(result.Binary)}
+			cmd := executeCommand(config.Options, "arm-none-eabi-gdb", args...)
```

Comment:
This is also Linux specific. Can you please use `LookupGDB` instead?

Reply:
I could use LookupGDB without problem, 'arm-none-eabi-gdb' was properly detected. 
(Only tested on Linux, with a couple of STM32 boards)